### ISSUE TITLE
How to choose the trained model for Kitti Testset?

### ISSUE THREAD

**Issue by WERush** [traveller59/second.pytorch#72]
Hi, 

Thanks for sharing the code. I am wondering how to choose the model for Kitti Testset, and have you trained the full model by using training set and validation set? Thanks.

**Comment by traveller59** [comment]
just choose models with best val performance...
No, the submitted model only trained with 50/50 split. Please don't submit result with other split.

**Comment by WERush** [comment]
Hi,
In your voxel_encoder part, you use 'VoxelFeatureExtractorV3' to extract the voxel features, and the function 'VoxelFeatureExtractorV3' only generates the mean points coordinates in voxels. It looks much simple, is it a key component for performance improvement? Thanks.

**Comment by traveller59** [comment]
The voxel size of SECOND-V1.5 is decreased to only (0.5, 0.5, 1), there is no need to use VFE (slow, use too much GPU memory). The key of performance improvement is high-resolution sparse data and large sparse convolution networks.

**Comment by pangsu0613** [comment]
> Did you try the training with all data (training and val data)?
> Because the official Kitti dataset consists of only training set (7481 images) and test set(7518 images), training with all data (training and validation set) is okay.

Hello, have you tried to train with all training data? Is the performance better? (I guess it would be difficult to evaluate, since the original validation data in the training set is also used for training) Thank you

**Comment by pangsu0613** [comment]
> Did you try the training with all data (training and val data)?
> Because the official Kitti dataset consists of only training set (7481 images) and test set(7518 images), training with all data (training and validation set) is okay.

@WERush Hello, have you tried to train with all training data? Is the performance better? (I guess it would be difficult to evaluate, since the original validation data in the training set is also used for training) Thank you

**Comment by WERush** [comment]
@pangsu0613  I do not try to train with full data. A possible solution is to use a smaller validation set (in order to incorporate more data into training set), however it is hard to select.

**Comment by liqisa** [comment]
Maybe Kitti Testset's label is not released. so Testset can't be used to train.
